fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/cpiom_b.rs

```diff
@@ -979,19 +979,21 @@ impl<C: PressurizationConstants> CabinPressureControlSystemApplication<C> {
     ) {
         let (adirs_airspeed, adirs_ambient_pressure) = self.adirs_values_calculation(adirs);
         self.exterior_airspeed = adirs_airspeed.unwrap_or_default();
+        let new_exterior_altitude: Length;
 
-        self.exterior_flight_altitude = if !self.is_initialised {
+        if !self.is_initialised && adirs_ambient_pressure.is_some() {
```

Comment:
Please assign `new_exterior_altitude` here.
Also just to make sure: this branch can only be taken when it's not initialised and the adirs are available.

Reply:
Yes you are right, this condition is not very useful, it was probably me trying to be overly safe when making sure there was no condition where the reference pressure was 0. I reverted these changes now.